### PR TITLE
Replace old global config with flask's config

### PR DESCRIPTION
**Pull request by richardzilincikPantheon**

### CODE REVIEW

**Comment by richardzilincikPantheon** [comment]
Converted to draft for now because I think I broke the sender

**Comment by richardzilincikPantheon** [comment]
Looks like I didn't

**Comment by richardzilincikPantheon** [comment]
Resolves #287

File: api/yangCatalogApi.py [hunk 1]
```diff
@@ -88,8 +103,96 @@ def __init__(self, import_name):
...
+        self.config['S-CONFD-CREDENTIALS'] = self.config.s_confd_credentials.strip('"').split()
+        self.config['DB-ES-AWS'] = True if self.config.db_es_aws == 'True' else False
+        if self.config.db_es_aws:
+            self.config['ES'] = Elasticsearch([self.config.es_host],
+                                              http_auth=(self.config.s_elk_credentials[0],
+                                                         self.config.s_elk_credentials[1]),
+                                              scheme="https", port=443)
+        else:
+            self.es = Elasticsearch([{'host': '{}'.format(self.config.db_es_host), 'port': self.config.db_es_port}])
+
```
Comment (SlavomirMazurPantheon) [inline api/yangCatalogApi.py:163]:
```
self.config['ES'] = Elasticsearch([{'host': '{}'.format(self.config.db_es_host), 'port': self.config.db_es_port}])
```

File: api/views/yangSearch/yangSearch.py [hunk 2]
```diff
@@ -84,11 +82,20 @@ def __init__(self, name, import_name, static_folder=None, static_url_path=None,
...
 
 
-app = YangSearch('yangSearch', __name__)
+bp = YangSearch('yangSearch', __name__)
 
 
+@bp.record
+def init_logger(state):
+    bp.LOGGER = log.get_logger('healthcheck', '{}/healthcheck.log'.format(state.app.config.d_logs))
+
```
Comment (SlavomirMazurPantheon) [inline api/views/yangSearch/yangSearch.py:91]: Wrong log filename

**Review by SlavomirMazurPantheon — CHANGES_REQUESTED** [review]

**Review by SlavomirMazurPantheon — APPROVED** [review]

**Pull request merged**